ansys/dpf/core/custom_fields_container.py

```diff
@@ -88,15 +104,23 @@ def shell_fields(self, timeid=None, complexid=None):
         return self.get_fields(label_space)
     
     def beam_fields(self, timeid=None, complexid=None):
-        """Returns a list of all the fields with beam element shapes.
-        If a timeid or a complexid (0 for real and 1 for imaginary) are given, 
-        the list of fields returned are beam fields for a given time and/or 
-        complex type
+        """Retrieve a list of all fields with beam element shapes.
         
+        You can filter the list of fields with beam element shapes based on
+        a given time, complex type, or both.
+        
+        Parameters
+        ----------
+        timeid : int, optional
+            Time ID for filtering fields with beam element shapes.
+        complexid : int, optional
+            Complex type ID for filtering fields with beam element shapes. 
+            0 is for real numbers, and 1 is for imaginary numbers.
```

Comment:
Thanks for adding this.  I know it's a copy/paste, but parameters should always be documented.